app/services/export_files/packager.rb

```diff
@@ -1,32 +1,37 @@
 module ExportFiles
   class Packager
 
-    attr_reader :ability, :package, :work
+    attr_reader :ability, :package, :work_id
 
     delegate :add_payload_file, to: :package
 
     WORK_PATH_MAX_LENGTH = 30
 
-    def initialize(package, work, ability:)
+    def initialize(package, work_id, ability:)
       @package = package
-      @work = work
+      @work_id = work_id
       @ability = ability
     end
 
     def build!
-      handle_work(work, PayloadFilePackager::PAYLOAD_PATH)
+      work_doc = SolrDocument.find(work_id)
```

Comment:
Consider setting `work_doc` as an instance variable lazily with a method instead of passing it around.

```ruby
def work_doc
  @work_doc ||= SolrDocument.find(work_id)
end
```

Reply:
Oops, I see there's recursion, so probably nevermind.